src/pages/en/developers/apis/4-realtime-api/vehicle-positions/digitransit-mqtt/index.md

```diff
@@ -53,25 +53,25 @@ This list contains cities or regions for which there is vehicle position data av
-| Hämeenlinna    | [Hameenlinna](https://api.digitransit.fi/routing-data/v3/waltti/Hameenlinna-gtfs.zip)   | [waltti](https://api.digitransit.fi/routing/v1/routers/waltti/index/graphql)
-| Lahti          | [Lahti](https://api.digitransit.fi/routing-data/v3/waltti/Lahti-gtfs.zip)               | [waltti](https://api.digitransit.fi/routing/v1/routers/waltti/index/graphql)
-| Vaasa          | [Vaasa](https://api.digitransit.fi/routing-data/v3/waltti/Vaasa-gtfs.zip)               | [waltti](https://api.digitransit.fi/routing/v1/routers/waltti/index/graphql)
-| Mikkeli        | [Mikkeli](https://api.digitransit.fi/routing-data/v3/waltti/Mikkeli-gtfs.zip)           | [waltti](https://api.digitransit.fi/routing/v1/routers/waltti/index/graphql)
+| Tampere        | [tampere](https://api.digitransit.fi/routing-data/v3/waltti/tampere-gtfs.zip)           | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Jyväskylä      | [LINKKI](https://api.digitransit.fi/routing-data/v3/waltti/LINKKI-gtfs.zip)             | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Lappeenranta   | [Lappeenranta](https://api.digitransit.fi/routing-data/v3/waltti/Lappeenranta-gtfs.zip) | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Joensuu        | [Joensuu](https://api.digitransit.fi/routing-data/v3/waltti/Joensuu-gtfs.zip)           | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Kuopio         | [Kuopio](https://api.digitransit.fi/routing-data/v3/waltti/Kuopio-gtfs.zip)             | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Turku          | [FOLI](https://api.digitransit.fi/routing-data/v3/waltti/FOLI-gtfs.zip)                 | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Oulu           | [OULU](https://api.digitransit.fi/routing-data/v3/waltti/OULU-gtfs.zip)                 | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Hämeenlinna    | [Hameenlinna](https://api.digitransit.fi/routing-data/v3/waltti/Hameenlinna-gtfs.zip)   | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Lahti          | [Lahti](https://api.digitransit.fi/routing-data/v3/waltti/Lahti-gtfs.zip)               | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Vaasa          | [Vaasa](https://api.digitransit.fi/routing-data/v3/waltti/Vaasa-gtfs.zip)               | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
+| Mikkeli        | [Mikkeli](https://api.digitransit.fi/routing-data/v3/waltti/Mikkeli-gtfs.zip)           | [waltti](https://api.digitransit.fi/routing/v2/routers/waltti/index/graphql)
 | Salo           | [Salo](https://api.digitransit.fi/routing-data/v3/waltti-alt/Salo-gtfs.zip)             | To be included
```

Comment:
These to be included things can be replaced with the matching routing endpoint for the routing data endpoint on the same row, so for instance https://api.digitransit.fi/routing/v2/routers/waltti-alt/index/graphql on this row. They were to be included because there wasn't otp1 end point for these but for otp2 there is.